src/Microsoft.Data.Analysis/DataFrame.cs

```diff
@@ -42,19 +42,36 @@ public DataFrame(IList<DataFrameColumn> columns)
             _columnCollection = new DataFrameColumnCollection(columns, OnColumnsChanged);
         }
 
+        /// <summary>
+        /// Returns the number of rows in this <seealso cref="DataFrame"/>
```

Comment:
`seealso cref` => `see cref`. This probably applies everywhere.

Reply:
```suggestion
        /// Returns the number of rows in this <see cref="DataFrame"/>.
```